config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users do
+    resources :posts do
+      resources :comments
+    end
+  end
```

Comment:
- [ ] It will be great to only set up the routes that are used in the application. Kindly don't expose unused routes. You can use rails methods `only` or `expect` to define the only used routes

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```